src/Catalog/Dnx/DnxMaker.cs

```diff
@@ -302,6 +458,13 @@ public static string GetRelativeAddressNupkg(string id, string version)
             return $"{normalizedVersion}/{id}.{normalizedVersion}.nupkg";
         }
 
+        private static string GetRelativeAddressIcon(string id, string version)
+        {
+            var normalizedVersion = NuGetVersion.Parse(version).ToNormalizedString();
+
+            return $"{normalizedVersion}/icon";
```

Comment:
Does client know where they are storing icons locally yet? Originally, this folder structure was supposed to mimic the V3 folder structure on disk.

Reply:
We already deviated from that with licenses. Those are stored as `/license` under appropriate packageid/version path. I am being consistent with licenses here.

Reply:
Sounds good.